modules/mobile/spec/requests/mobile/v0/user/addresses_spec.rb

```diff
@@ -8,14 +8,19 @@
   let!(:user) { sis_user }
 
   before do
-    allow(Flipper).to receive(:enabled?).with(:mobile_v2_contact_info, instance_of(User)).and_return(false)
-    allow(Flipper).to receive(:enabled?).with(:va_v3_contact_information_service, instance_of(User)).and_return(false)
+    allow(Flipper).to receive(:enabled?).with(:mobile_v2_contact_info, instance_of(User)).and_return(true)
+    allow(Flipper).to receive(:enabled?).with(:va_v3_contact_information_service, instance_of(User)).and_return(true)
     allow(Flipper).to receive(:enabled?).with(:remove_pciu, instance_of(User)).and_return(false)
+    Timecop.freeze(Time.zone.parse('2024-08-27T18:51:06.012Z'))
   end
 
-  describe 'update endpoints', :skip_va_profile_user do
+  after do
+    Timecop.return
+  end
+
+  describe 'update endpoints' do
     let(:address) do
-      address = build(:va_profile_address, vet360_id: user.vet360_id)
+      address = build(:va_profile_v3_address, :v2_override)
```

Comment:
` build(:va_profile_v3_address) ` is the correct factory. The addresses do not match in the VCR requests. Please update all the VCR addresses to match [va_profile_v3_address](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/factories/va_profile/v3/addresses.rb#L4)